src/resources/user.yaml

```diff
@@ -115,3 +115,37 @@ actions:
     notes:
       - |
         Results are sorted by user ID.
+
+  - name: add
+    class: add-user
+    method: POST
+    path: "/workspaces/%s/addUser"
+    params:
+      - name: workspace
+        <<: *Param.WorkspaceId
+        required: true
+        comment: The workspace or organization to invite the user to.
+      - name: user
+        <<: *Param.User
+        required: true
+        comment: The email address to send the invite to.
+    comment: |
+      The user can be referenced by their globally unique user ID or their email address.
+      Returns the full user record for the invited user.
+
+  - name: remove
+    class: remove-user
+    method: POST
+    path: "/workspaces/%s/removeUser"
+    params:
+      - name: workspace
+        <<: *Param.WorkspaceId
+        required: true
+        comment: The workspace or organization to invite the user to.
+      - name: user
+        <<: *Param.User
+        required: true
+        comment: The user to invite
```

Comment:
This comment seems off. Also note that comments written here will overwrite the comment defined on `Param.User`. That may be what you want, but just FYI.